AirCasting/Models/Session+Mock.swift

```diff
@@ -15,8 +15,25 @@ extension SessionEntity {
         let session: SessionEntity = try! context.newOrExisting(uuid: SessionUUID(rawValue: "mock")!)
         session.type = .mobile
         // ...
+                
         return session
     }
 }
 
+extension MeasurementStreamEntity {
```

Comment:
Is there a reason why it's not in a separate file?

Reply:
Well, I guess I wanted to have mocked session and mocked stream in one file. 
Do you think I should move it elsewhere?

Reply:
It's not very intuitive to look for a MeasurementStreamEntity extension in SessionMock file